.circleci/config.yml

```diff
@@ -52,11 +54,23 @@ jobs:
       - run:
           name: Build monorepo
           command: yarn build
+      - run:
+          name: Download and verify Codecov uploader
+          command: |
+            curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step
```

Comment:
Let's put the codecov downloader inside `ci-builder` directly. This way we avoid having to download and install it on every build.

Reply:
OK, I've added that in a [new commit](https://github.com/ethereum-optimism/optimism/pull/3348/commits/2b6e5773494eec3c23f9f0e6a74718e86611f6eb). 

To avoid failures caused by running CI before a new release is made, I've also made it so that the upload step [will always pass](https://github.com/ethereum-optimism/optimism/pull/3348/commits/8c902ba18e142f340d700418c0356bbe09fb9796).